src/game_theory/sprague-grundy-nim.md

```diff
@@ -114,13 +114,13 @@ Since the game is acyclic, sooner or later the current player won't be able to u
 
 ### Sprague-Grundy theorem
 
-Let's consider a state $v$ of a two-player impartial game and let $v_i$ be the states reachable from it (where $i \in \{ 1, 2, \dots, k \} , k \ge 0$).
+Let's consider a state $v$ of a two-player impartial game and let $v_i$ be the states reachable from it (where $i \in \\{ 1, 2, \dots, k \\} , k \ge 0$).
 To this state, we can assign a fully equivalent game of Nim with one pile of size $x$.
 The number $x$ is called the Grundy value or nim-value of state $v$.
 
 Moreover, this number can be found in the following recursive way:
 
-$$ x = \text{mex} \{ x_1, \ldots, x_k \}, $$
+$$ x = \text{mex}\ \\{ x_1, \ldots, x_k \\}, $$
```

Comment:
Same here. And the first backslash + space just gives a little bit of horizontal space.

Reply:
Same things also on the other changes of the commit.

Reply:
Again here -- my change is from `\{` to `\\{`, not the other way around. And I thought it looks nicer with that extra space between mex and the brackets (since there are no parentheses)